org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/wizards/ParameterTableInspector.java

```diff
@@ -0,0 +1,122 @@
+ * endorse or promote products derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.openjdk.jmc.console.ext.agent.manager.wizards;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jface.viewers.ColumnLabelProvider;
+import org.eclipse.jface.viewers.TableViewer;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.openjdk.jmc.console.ext.agent.manager.model.ICapturedValue;
+import org.openjdk.jmc.console.ext.agent.manager.model.IMethodParameter;
+import org.openjdk.jmc.console.ext.agent.manager.model.IMethodReturnValue;
+import org.openjdk.jmc.console.ext.agent.manager.model.INamedCapturedValue;
+import org.openjdk.jmc.ui.column.ColumnBuilder;
+import org.openjdk.jmc.ui.column.ColumnManager;
+import org.openjdk.jmc.ui.column.IColumn;
+import org.openjdk.jmc.ui.misc.OptimisticComparator;
+import org.openjdk.jmc.ui.misc.TreeStructureContentProvider;
+
+public class ParameterTableInspector {
+
+	public static final String PARAMETER_TREE_NAME = "EventEditingWizardParameterPage.ParamtersTree"; //$NON-NLS-1$
+	private static final String INDEX_COLUNM_ID = "index";
+	private static final String INDEX_COLUNM_NAME = "Index";
+	private static final String NAME_COLUNM_ID = "name";
+	private static final String NAME_COLUNM_NAME = "Name";
+	private static final String DESCRIPTION_COLUNM_ID = "description";
+	private static final String DESCRIPTION_COLUNM_NAME = "Description";
+	private final TableViewer viewer;
+
+	private final ColumnLabelProvider nameLabelProvider = new ColumnLabelProvider() {
+		@Override
+		public String getText(Object element) {
+			if (element instanceof ICapturedValue) {
+				return ((INamedCapturedValue) ((ICapturedValue) element)).getName();
+			}
+			return element.toString();
+		}
+	};
+
+	private final ColumnLabelProvider indexLabelProvider = new ColumnLabelProvider() {
+		@Override
+		public String getText(Object element) {
+			if (element instanceof IMethodParameter) {
+				return (String.valueOf(((IMethodParameter) element).getIndex()));
```

Comment:
redundant parentheses

```suggestion
				return String.valueOf(((IMethodParameter) element).getIndex());
```